.github/workflows/build-rock.yaml

```
@@ -13,35 +13,27 @@ on:
         default: "latest/stable"
         type: string
       source-branch:
-        description: Github branch to checkout. If blank, it will use default values.
-        default: '' # For default behaviour, see https://github.com/actions/checkout/tree/v4/#usage
+        description: "Github branch to checkout. If blank, it will use default values."
```

nitpick: Slightly more clear sentencing
```suggestion
        description: "Github branch to checkout. If blank, it will use the default branch of the repository."
```